backend/crypt/crypt.go

```diff
@@ -45,7 +45,7 @@ func init() {
 					Help:  "Very simple filename obfuscation.",
 				}, {
 					Value: "off",
-					Help:  "Don't encrypt the file names.  Adds a \".bin\" extension only.",
+					Help:  "Don't encrypt the file names. Adds a \".bin\" extension only.",
```

Comment:
This should probably have a `\n\n` in

Reply:
Well, this is in the example part - the help/description of an the example value. I don't think these are being processed like the option help strings, i.e. extract first line, and are never shown on command line, so I tried to leave these as is - just removed an additional space on this one.